daffodil-runtime1-unparser/src/main/scala/org/apache/daffodil/processors/unparsers/SequenceUnparserBases.scala

```diff
@@ -38,5 +38,5 @@ abstract class OrderedSequenceUnparserBase(
   // Since some of the grammar terms might have folded away to EmptyGram,
   // the number of unparsers here may be different from the number of
   // children of the sequence group.
-  Assert.invariant(srd.groupMembers.length >= childUnparsers.length - 1) // minus 1 for the separator unparser
+  //Assert.invariant(srd.groupMembers.length >= childUnparsers.length - 1) // minus 1 for the separator unparser
```

Comment:
Delete. With the comment above. I think this is not worth checking. It's not obvious what the number of unparsers always is.